src/plexil-adapter/OwInterface.cpp

```diff
@@ -621,10 +577,32 @@ void OwInterface::initialize()
        subscribe("/pt_faults_status", qsize,
                 &OwInterface::antennaFaultCallback, this)));
 
-    ROS_INFO ("Waiting for action servers...");
-    m_guardedMoveClient.reset(new GuardedMoveActionClient("GuardedMove", true));
-    m_guardedMoveClient->waitForServer();
-    ROS_INFO ("Action servers available.");
+    m_guardedMoveClient.reset(new GuardedMoveActionClient(Op_GuardedMove, true));
+    m_unstowClient.reset(new UnstowActionClient(Op_Unstow, true));
+    m_stowClient.reset(new StowActionClient(Op_Stow, true));
+    m_grindClient.reset(new GrindActionClient(Op_Grind, true));
+    m_digCircularClient.reset(new DigCircularActionClient(Op_DigCircular, true));
+    m_digLinearClient.reset(new DigLinearActionClient(Op_DigLinear, true));
+    m_deliverClient.reset(new DeliverActionClient(Op_Deliver, true));
```

Comment:
since we have upgraded to C++14 you should be able to use `std::make_unique` which is more robust that doing reset(new ...) to set the a **unique_ptr**

Reply:
Created a more general and unassigned Jira OW-669 for this and also upgrading the raw pointers, which you can take if you like.  I think you were the one who introduced unique_ptr and reset() !

Reply:
Yes, but back then we were still compiling against C++11 standard which lacked the `std::make_unique` function.

Reply:
Oh, I didn't know that.  I thought we were 14-compliant since the beginning of the project.